pom.xml

```diff
@@ -46,16 +46,17 @@
 
     <groupId>fish.payara.extras</groupId>
     <artifactId>payara-upgrade-tool</artifactId>
-    <version>2.0-SNAPSHOT</version>
+    <version>2.0</version>
     <packaging>glassfish-jar</packaging>
     <name>Payara Upgrade Tool</name>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <payara.version>5.36.0</payara.version>
+        <payara.version>6.1.0-SNAPSHOT</payara.version>
```

Comment:
Do we actually need to use 6.1.0?
Could we not use the already published 6.0.0?

Reply:
done